<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1218</width>
    <height>854</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{ 
        background-color: #C9C8C8}
    </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>1221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom: 1px solid black; 
background-color:#DED9D9</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,1,12,1,0">
     <property name="spacing">
      <number>15</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>8</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="home_button">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="history_button">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="exit_button">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>8</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>1201</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,1">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="maximumSize">
            <size>
             <width>160</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size:9pt;
 font-weight:500;</string>
           </property>
           <property name="text">
            <string> Patient Details</string>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="1" column="0">
            <widget class="QLabel" name="label_6">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>32</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"> margin: 5px; 
font-size:8pt;
</string>
             </property>
             <property name="text">
              <string>Patient ID</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QTextEdit" name="patient_id_text">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>28</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>32</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"> margin: 5px; 
 font-size:8pt;
</string>
             </property>
             <property name="text">
              <string>Patient Name</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QTextEdit" name="patient_name_text">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>28</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_7">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>28</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"> margin: 5px; 
font-size:8pt;</string>
             </property>
             <property name="text">
              <string>Birth Date</string>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QTextEdit" name="patient_bd_text">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>28</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="maximumSize">
              <size>
               <width>160</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font-size:9pt;
 font-weight:500;</string>
             </property>
             <property name="text">
              <string>Study Description</string>
             </property>
             <property name="margin">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextBrowser" name="textBrowser"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="maximumSize">
            <size>
             <width>160</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size:9pt;
 font-weight:500;</string>
           </property>
           <property name="text">
            <string>Selected Algorithm</string>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="list_used_algo"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="maximumSize">
            <size>
             <width>160</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size:9pt;
 font-weight:500;</string>
           </property>
           <property name="text">
            <string>Alternative Algorithms</string>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="list_unused_algo"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>60</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="save_button">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ok_button">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>1201</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:9pt;
font-weight:500;
text-align:center;

</string>
    </property>
    <column>
     <property name="text">
      <string>Organs</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1218</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
